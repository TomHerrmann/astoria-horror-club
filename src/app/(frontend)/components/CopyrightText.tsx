/**
 *
 * @returns Copyright information for AHC
 */
export default function CopyrightText() {
  return <p className="m-[0px]">Â© 2021 Astoria Horror Club</p>
}
