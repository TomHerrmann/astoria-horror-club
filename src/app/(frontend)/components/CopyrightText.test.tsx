import { describe, it, expect } from 'vitest'
import { render, screen } from '@testing-library/react'
import CopyrightText from './CopyrightText'

describe('CopyrightText', () => {
  it('matches snapshot', () => {
    const { container } = render(<CopyrightText />)
    expect(container).toMatchSnapshot()
  })

  it('renders the copyright text correctly', () => {
    render(<CopyrightText />)
    expect(screen.getByText('Â© 2021 Astoria Horror Club')).toBeDefined()
  })
})
