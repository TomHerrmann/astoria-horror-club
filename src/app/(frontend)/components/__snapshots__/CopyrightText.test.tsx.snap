// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CopyrightText > matches snapshot 1`] = `
<div>
  <p
    class="m-[0px]"
  >
    Â© 2021 Astoria Horror Club
  </p>
</div>
`;
